### DISCUSSION TITLE
Flash/blink when using Flameshot

### DISCUSSION THREAD

**Discussion by blirp** [flameshot-org/flameshot#2074] [Q&A]
There seems to be a flash/blink to white if I hover the mouse over the 'save'-button. How do I turn that off? It is really very annoying.

**Comment by mmahmoudian** [comment]
Can you record a quick video of what happens?

**Reply by blirp** [reply]
Weird having multiple screen capture programs running at the same time, but I made an attempt:

https://user-images.githubusercontent.com/962761/141866008-e4d7e489-6e75-47ad-98ae-df460099efd3.mp4

**Reply by mmahmoudian** [reply]
I tried to see the video with iOS Firefox, Linux Firefox, and the Github iOS app and all of them failed to play the video:

![image](https://user-images.githubusercontent.com/390889/142016272-052434d8-43ee-4ec6-aeb9-c0b2ed83bbce.png)


Please try uploading the mp4 again or uploading it to a service like https://0x0.st/ or dropbox

**Reply by blirp** [reply]
Yeah, noticed that. Worked with Chromium on Ubuntu, though.

**Comment by blirp** [comment]
This seems to work on my Firefox, at least.

https://user-images.githubusercontent.com/962761/142067650-08a0a01e-0a96-4115-ab27-2dfc99a011b0.mp4

**Reply by mmahmoudian** [reply]
1. is it only one the save button?
2. Does it happen when you click or in only happens when you hover with mouse?
3. What is your computer's specs? OS, monitors, GPU, ...

**Comment by blirp** [comment]
1. No, it seems to be all available buttons.
2. It happens on hover. It actually seems to be related to the display of the hover hint. That is, if I move the cursor quickly from one button to another, the hover hint text changes, but I don't see any blinking. If I move slower, there is a huge blink when the hover hint reappears
3. Specs: 
![specs](https://user-images.githubusercontent.com/962761/142781933-055f0eb2-e0ac-4c5a-be12-b97a3d7c84d2.png)

**Reply by mmahmoudian** [reply]
This type of thing might be compositor issue as well. I don't know what is the best way to approach this issue though. Try installing/updating your compositor which I assume is compton.
